src/dataHelper.js

```diff
@@ -95,16 +135,26 @@ export const findData = (lang, json, files, media, doc) => {
         }
         break;
       case 'paragraph--card':
+        let cards = convertCardsFromDrupal([item], false);
```

Comment:
In general, 'const' over 'let' would be better for code readability: tells that this won't be reassigned.